irr/src/CImageLoaderPNG.cpp

```diff
@@ -18,14 +18,20 @@ namespace video
 // PNG function for error handling
 static void png_cpexcept_error(png_structp png_ptr, png_const_charp msg)
 {
-	os::Printer::log("PNG fatal error", msg, ELL_ERROR);
+	io::IReadFile *file = reinterpret_cast<io::IReadFile *>(png_get_error_ptr(png_ptr));
+	core::stringc logmsg = core::stringc("PNG fatal error for ") + file->getFileName()
```

Comment:
I would avoid introducing new uses of stringc/stringw

Reply:
Ok, done. (Was doing this because `file->getFileName()` is a path aka stringc.)